docs/week-5/classes-objects.md

```diff
@@ -2,16 +2,16 @@
 
 You've been using Java classes since week 1, with our first Hello World program, but what are classes, really?
 
-**Classes** are Java's way of grouping variables, methods, and other classes together. 
+**Classes** are Java's way of grouping related variables and methods together. 
```

Comment:
aw but you can put classes in classes

Reply:
I know but this is of limited utility to a new programmer and likely to cause confusion.